_harp/academy/_prerequisites.md

```diff
@@ -1,44 +1,26 @@
-# Course prerequisites
+## Other recommended materials
 
-We have chosen these resources as our entrance criteria. If you are not comfortable coding at this standard by the time the course starts you may struggle. These tasks are part of how we will measure whether you are a good candidate for the course as well as whether you can learn in the collaborative environment here. You do not need to have completed these prerequisites before you send in your application, but you should at least have made a good start on them. 
```

Comment:
I think this is actually still a good warning. I would delete the first sentence (this isn't entrance criteria) but the warning that you may struggle if you aren't comfortable with these is actually very useful.